chromium_src/ui/native_theme/native_theme_android.cc

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2019 The Brave Authors
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "../../../../../ui/native_theme/native_theme_android.cc"
+
+#if !defined(USE_AURA)
```

Comment:
nit: this define guard is not needed because this file is only included for android build.

Reply:
Good point, will fix.